core-field-editors/README.md

```diff
@@ -0,0 +1,32 @@
+Contentful Core Field Editors
+=============================
+
+This folder contains the source code to some of the Contentful Web App’s builtin
+field editors. The code targets the [Contentful UI Extension
+API][ui-extension-sdk]. This allows you to modify the source code, upload the
+editors as extensions to Contentful and use them in our App.
+
+To do this clone the repository and install the field editor dependencies.
+~~~
+git clone https://github.com/contentful/extensions.git contentful-extensions
+cd contentful-extensions/core-field-editors
+npm install
+npm install -g contentful-extension-cli
+~~~
+
+Move to the field editor folder and bundle the source code.
+~~~
+cd src/textarea
+../../bin/bundle
+~~~
+This will create the `editor.dist.html` file including the bundled and inlined
+code.
+
+Finally, create the extension from the `extension.json` definition and the HTML
+file.
+~~~
+export CONTENTFUL_MANAGEMENT_ACCESS_TOKEN=<MY_TOKEN>
+contentful-extension create --space-id <MY_SPACE_ID>
+~~~
+
```

Comment:
I'd add: "You can now use the component by following the same steps as with any other [UI extension](https://www.contentful.com/blog/2016/07/06/ui-extensions-sdk/)"